cursive-core/src/views/dialog.rs

```diff
@@ -555,15 +565,26 @@ impl Dialog {
         match result {
             EventResult::Ignored => {
                 match event {
```

Comment:
You could `match (event, self.button_orientation) {`, then match on `(Event::Key(Key::Up), Orientation::Horizontal)`, which could be easier than adding `if self.button_orientation == ...` every time.

Reply:
Done